cli/magic-create.ts

```diff
@@ -156,8 +156,8 @@ async function processCreateOptions(options: any): Promise<void> {
     {
       type: "multiselect",
       name: "sagemakerModels",
-      message:
-        "Which SageMaker Models do you want to enable (enter for None, space to select)",
+      hint: "SPACE to select, ENTER to confirm selection",
```

Comment:
Can we add "Optional" so it doesn't look like you have to choose one?

Reply:
What about `"Do you want to enable any SagemMaker models?"`

Reply:
Yes